numpy/core/tests/test_multiarray.py

```diff
@@ -4333,7 +4333,41 @@ def test_object_argmax_with_NULLs(self):
         assert_equal(a.argmax(), 3)
         a[1] = 30
         assert_equal(a.argmax(), 1)
+    
+    def test_np_argmax_keepdims(self):
 
+        sizes = [(3,), (3, 2), (2, 3),
+                 (3, 3), (2, 3, 4), (4, 3, 2),
+                 (1, 2, 3, 4), (2, 3, 4, 1),
+                 (3, 4, 1, 2), (4, 1, 2, 3)]
```

Comment:
Would be nice to parametrize the shapes.  (look for `pytest.mark.parametrize` for examples)